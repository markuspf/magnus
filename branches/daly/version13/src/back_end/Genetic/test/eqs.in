<a,b>
(a b)^5 x (b a)^5,


