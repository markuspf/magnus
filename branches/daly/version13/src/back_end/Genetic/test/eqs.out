<a,b>
a^N x b^N = 1
Minimal solution has length 2N

length: num of generations

10: 32,28,62,16,12,7,25,6,7,47
20: 38,38,59,46,76,243,96,55,123,64
30: 109,108,151,155,89,77,220,175,183,260
40: 158,213,151,208,260,237,306,213,290,242
50: 190,248,250,588,198,365,437,144,154,474
60: 240,330,245,427,179,223,310,768,479,290

100: 401,888,852,602,612,871,1260,417,729,506
200: 3006,1118,613,1970,3165,3222,2300,3375,3496,4101